packages/api/prisma/schema.prisma

```diff
@@ -36,3 +38,9 @@ model Verse {
 
   @@unique([chapterId, number])
 }
+
+model Language {
+  id   String @id @default(dbgenerated("generate_ulid()")) @db.Uuid
+  code String @unique
```

Comment:
Two comments here:
1. I'm a big fan of UUIDs for primary keys for a variety of reasons, but they have a problems with indexing. [A ULID](https://blog.daveallie.com/ulid-primary-keys) is more deterministic because it has a timestamp which solves some of these problems.
2. Even though it is unlikely for a code to be changed, I'm still inclined to have an opaque ID to use for foreign keys.